docs/index.md

```diff
@@ -50,21 +64,17 @@ df = spark.read.parquet("s3a://my-repo/main-branch/collections/foo/")
 
 You can use the same methods and syntax you are already using to read and write data when using a lakeFS repository. This simplifies the adoption of lakeFS - minimal changes are needed to get started, making further changes an incremental process.
 
```

Comment:
I think an internal link to the documentation of the integrations we have would be great here (I should have added it before already :-) )

Reply:
added